docs/src/main/sphinx/release/release-352.md

```diff
@@ -0,0 +1,88 @@
+
+## Web UI
+
+* Support JWK with certificate chain for OAuth2 authorization. ({issue}`6428`)
+* Optionally verify JWT audience (`aud`) field for OAuth2 Web UI authentication. ({issue}`6501`)
+* Add scopes to `OAuth2` configuration. ({issue}`6580`)
+* Prevent from replay attacks in `OAuth2` by using `nonce` cookie when `openid` scope is requested. ({issue}`6580`)
+
+## JDBC Driver
+
+* Support user impersonation when using password-based authentication. This is configured 
+  using `sessionUser` connection URL parameter. Previously user impersonation was possible
+  only with other authentication mechanisms. ({issue}`6549`)
+* Add OAuth2 authentication support.({issue}`6576`)
+
+## Docker Image
+
+* Remove support for configuration directory `/usr/lib/trino/etc`. The configuration should be provided in `/etc/trino`.
+
+## CLI
+
+* Support user impersonation when using password-based authentication. This is configured
+  using `--session-user` command line parameter. Previously user impersonation was possible 
+  only with other authentication mechanisms. ({issue}`6567`)
+
+## BigQuery Connector
+
+* Fix query failure when executing `count(*)` without `WHERE` clause for views. ({issue}`6706`)
+
+## Hive Connector
+
+* Support reading timestamp with microsecond or nanosecond precision (as configured with 
+  `hive.timestamp-precision` property) nested within `array`, `map` or `struct` data types. ({issue}`5195`)
+* Support reading from table in Sequencefile format that uses LZO compression. ({issue}`6452`)
+* Expose AWS HTTP Client stats via JMX. ({issue}`6503`)
+
+## Iceberg Connector
+
+* Optimize cardinality-insensitive aggregations (max, min, distinct, approx_distinct) over 
+  identity partition columns with `optimizer.optimize-metadata-queries` config property 
+  or `optimize_metadata_queries` session property. ({issue}`5199`)
+* Provide  `use_file_size_from_metadata` catalog session property and `iceberg.use-file-size-from-metadata` 
+  config property to fix query failures on tables with wrong file sizes stored in the metadata. ({issue}`6369`)
+ 
+## Kinesis Connector
+
+* Support GZIP message compression. ({issue}`6442`)
+
+## MySQL Connector
+
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
+* Improve performance for certain queries using `stddev_pop`, `stddev_samp`, `var_pop`, `var_samp` aggregation 
+  functions by pushing the aggregation and predicates computation into the remote database. ({issue}`6673`)
+
+## PostgreSQL Connector
+
+* Improve performance for certain complex queries involving aggregation and predicates (e.g. `HAVING` clause)
+  by pushing the aggregation and predicates computation into the remote database. ({issue}`6667`)
+
+## Redshift Connector
+
+* Use dedicated Redshift JDBC driver to access Redshift. As a result, `connection-url` in catalog 
```

Comment:
remove "dedicated"

Reply:
replace with "the" ?

Reply:
I'm not even sure we should mention the fact that we're using a different JDBC driver at all. All we need to mention is the user-visible effect.

Reply:
@martint we should

- as an explanation why we changed the URL in breaking manner
- should there be any user visible consequences that we do not see (like type mapping), users should be heads-up'ed

Reply:
> should there be any user visible consequences that we do not see 

Are there?

Reply:
I am not aware of any other, but potentially you can put some stuff in JDBC url now that you couldn't before (or vice versa)

Reply:
The redshift jdbc driver is a fork of the postgresql one so negative changes are unlikely but we definitely have to add this note since the old JDBC URL will not continue to work.